packages/typegpu/src/data/numeric.ts

```diff
@@ -120,4 +158,11 @@ const f32Cast = (v: number | boolean) => {
   }
   return v;
 };
+/**
+ * Creates a new 32-bit float schema representing a single WGSL f32 value.
+ * Can also be used to cast a value to an f32.
```

Comment:
Since it by itself is a schema, rather than a function that creates a schema, I would make that clear in this doc string.

```suggestion
 * A schema that represents a 32-bit float value. (equivalent to `f32` in WGSL)
 * 
 * Can also be called to cast a value to an f32.
```